actions/src/sema4ai/actions/chat/_client.py

```diff
@@ -211,7 +211,8 @@ def set_bytes(
                 # Add the file to the request after the form data (the order
                 # of the fields is important -- the "key" field must
                 # be the first field).
-                fields.append(("file", content))
+                filename = "file"
```

Comment:
Actually, the `filename` is in the namespace, so, maybe it should be `use_filename = filename`.